--- XOM/build.xml	2009-03-08 20:42:21.000000000 +0100
+++ XOM/build.xml-gil	2009-03-08 20:43:55.000000000 +0100
@@ -368,7 +368,11 @@
            encoding="UTF-8"
            includes="nu/xom/tools/*">
       <classpath>
-        <pathelement path="${compile.class.path}"/>
+        <pathelement location="${xml-apis.jar}"/>
+        <pathelement location="${parser.jar}"/>
+        <pathelement location="${normalizer.jar}"/>
+        <pathelement location="${junit.jar}"/>
+        <pathelement location="${servlet.jar}"/>
         <pathelement path="${tagsoup.jar}"/> 
       </classpath>
     </javac>
