--- build.xml	2010-01-18 14:04:34.000000000 +0100
+++ build.xml-gil	2010-01-18 14:19:43.000000000 +0100
@@ -382,6 +382,7 @@
     <mkdir dir="${build.javadocs}"/>
     
     <javadoc packagenames="nu.xom.xslt, nu.xom.canonical, nu.xom.xinclude, nu.xom, nu.xom.converters"
+             additionalparam="-J-Xss1m"
              sourcepath="${build.src}"
              destdir="${build.javadocs}"
              author="true"
