--- build.xml	2011-09-25 13:58:27.989709164 +0200
+++ build.xml-gil	2011-09-25 13:59:58.583714735 +0200
@@ -392,7 +392,8 @@
              bottom="Copyright 2002-${year} &lt;a href='http://www.elharo.com/'&gt;Elliotte Rusty Harold&lt;/a&gt;
 			 &lt;br /&gt; &lt;a href='&#109;&#97;&#105;&#108;&#116;&#111;&#58;elharo%40metalab%2Eunc%2Eedu?Subject=XOM'&gt;elharo&#64;metalab&#46;unc&#46;edu&lt;/a&gt;">
       <classpath refid="test.class.path"/>
-      
+      <link href="${j2se.api}"/>
+      <link href="${junit.api}"/> 
     <fileset dir="${build.src}" defaultexcludes="yes">
       <include name="nu/xom/tests/XOMTestCase.java" />
     </fileset>
@@ -446,7 +447,8 @@
              bottom="Copyright 2002-${year} &lt;a href='http://www.elharo.com/'&gt;Elliotte Rusty Harold&lt;/a&gt;
 			 &lt;br/&gt; &lt;a href='mailto:elharo@metalab.unc.edu?Subject=XOM'&gt;elharo@metalab.unc.edu&lt;/a&gt;">
       <classpath refid="test.class.path"/>
-      
+      <link href="${j2se.api}"/>
+      <link href="${junit.api}"/> 
     <fileset dir="${build.src}" defaultexcludes="yes">
       <include name="nu/xom/tests/XOMTestCase.java" />
     </fileset>
